scripts/dock

```diff
@@ -129,5 +108,18 @@ export DOCKER_HOST=tcp://${DOCK_IP}:2377
 export DOCKER_IP=${DOCK_IP}
 
 if [ -z $DOCKERUTILS_DISABLE_PROMPT ]; then
-  update_dock_prompt
+    # Update command line prompt to reflect docked condition.
+    # Python virtual environment prompt (or lack therof) should remain unchanged.
+
+    _DOCK_MONIKER="[dock:$DOCK_MONIKER] "
```

Comment:
Might be nice to have this environment variable exposed with the prompt disabled, this would allow people to have their own PS1 in place to put the value where they would want it.  (just declare and export above the if statement).

Reply:
Good catch... made the change